string status
float32 current_x
float32 current_y
float32 distance_to_object
